kani-compiler/src/kani_middle/attributes.rs

```diff
@@ -1034,10 +1034,9 @@ fn attr_kind(tcx: TyCtxt, attr: &Attribute) -> Option<KaniAttributeKind> {
                     .intersperse("::")
                     .collect::<String>();
                 KaniAttributeKind::try_from(ident_str.as_str())
-                    .map_err(|err| {
+                    .inspect_err(|&err| {
                         debug!(?err, "attr_kind_failed");
                         tcx.dcx().span_err(attr.span, format!("unknown attribute `{ident_str}`"));
-                        err
                     })
```

Comment:
How is this change related to the others?

Reply:
Clippy enforced these, seems to be a part of the toolchain upgrade to clippy itself?

Reply:
Ok, thank you for clarifying!